Source/WebInspectorUI/UserInterface/Models/CSSStyleDeclaration.js

```diff
@@ -113,6 +115,22 @@ WI.CSSStyleDeclaration = class CSSStyleDeclaration extends WI.Object
     get locked() { return this._locked; }
     set locked(value) { this._locked = value; }
 
+    get useVariablesGrouping() { return this._useVariablesGrouping; }
```

Comment:
Style: we only do single line `get` or `set` if both the `get` and `set` are single line (or if there is only a `get`)